src/utils/theGraph.js

```diff
@@ -356,6 +359,7 @@ const completeQueries = {
                 return {
                   ...vault,
                   isMinionModule,
+                  minQuorum: minion.minQuorum,
```

Comment:
Added this as the VaultsAPi doesn't return this property

Reply:
Good catch. Will need to look into that.